### DISCUSSION TITLE
What is the default admin user? Where to run the commands?

### DISCUSSION THREAD

**Discussion by gittbuh** [ArchiveBox/ArchiveBox#872] [Q&A] [answered]
Hi all,

Just installed with docker compose. But how to login or create a new admin user? I run the command "archivebox help" inside the archivebox folder but I get "-bash: archivebox: command not found".

Thank you.

**Comment by mekberg** [comment, marked as answer]
Just tried out archivebox and ran into the same issue. Apparently, when installing the docker version, you have to use "docker-compose run archivebox" while inside the archivebox folder. So to run the "help" command, that would be

`docker-compose run archivebox help`
